paperqa/prompts.py

```diff
@@ -13,42 +10,37 @@
     "{text}\n\n"
     "Excerpt from {citation}\n"
     "Question: {question}\n"
-    "Relevant Information Summary:",
+    "Relevant Information Summary:"
 )
 
-qa_prompt = PromptTemplate(
-    input_variables=["context", "answer_length", "question"],
-    template="Write an answer ({answer_length}) "
-    "for the question below based on the provided context. "
+qa_prompt = (
+    "Write an answer ({answer_length}) "
+    "for the question below based on the provided context. Ignore irrelevant context. "
     "If the context provides insufficient information and the question cannot be directly answered, "
     'reply "I cannot answer". '
     "For each part of your answer, indicate which sources most support it "
     "via valid citation markers at the end of sentences, like (Example2012). \n"
```

Comment:
Could you swap the citation to be wikicrow style here:

(Qiu2020aarFDC pages 3-3)

from completion prompt: https://github.com/Future-House/WikiCrow

Reply:
Put latest prompts in